routes.js

```diff
@@ -0,0 +1,16 @@
+const express = require('express');// pass passport for configuration
+const router = express.Router();
+var User = require('./model/usermodel');
+
+router.get('/alluser',(req,res,next)=>{
+    User.find(function(err,user){
```

Comment:
/users instead of /alluser
Also, is this tested code? I think it should be `User.find({}, function(req,res,next)=>{`

Reply:
this works fine tested using postman

Reply:
Great!